db/citations.yml

```diff
@@ -10264,15 +10264,15 @@ yang;aca21:
   facility: ''
   grant: dmref15, ornlgo17
   journal: Acta Crystallogr. A
-  month: tbd
+  month: Jan
   nb: art
-  note: to be published
-  number: ''
+  note: ''
```

Comment:
it is safter to set this to `optnote` instead of `note` so it doesn't do any funny white-spacy thing on typesetting (not sure about that, I guess we should test it to see what happens with an empty string)